BenchmarkComponent/BenchmarkComponent.h

```diff
@@ -24,11 +26,46 @@ namespace winrt::BenchmarkComponent::implementation
         void DefaultBoolProperty(bool val);
         void DefaultDoubleProperty(double val);
     };
+
+    struct WrappedClass : WrappedClassT<WrappedClass>
+    {
+        WrappedClass();
+
+        int32_t DefaultIntProperty();
+        void DefaultIntProperty(int32_t val);
+    };
+
+    struct ClassWithMarshalingRoutines : ClassWithMarshalingRoutinesT<ClassWithMarshalingRoutines>
+    {
+        ClassWithMarshalingRoutines();
+
+        hstring DefaultStringProperty();
+        void DefaultStringProperty(hstring val);
+
+        /*Windows::Foundation::IInspectable DefaultWrappedClassProperty();
+        void DefaultWrappedClassProperty(Windows::Foundation::IInspectable val);*/

```

Comment:
These look to be commented out.  Remove or should they be there?

Reply:
I shall remove this.